landoapi/repos.py

```diff
@@ -266,6 +296,18 @@ def phab_identifier(self) -> str:
             "/1.0/firefox_versions.json",
             autoformat_enabled=True,
         ),
+        # Try uses `mozilla-unified` as the `pull_path` as using try
```

Comment:
Why did you choose `mozilla-unified` here? (vs say `autoland`, or `mozilla-central`?) Maybe just add a bit about that in the comment.